<!DOCTYPE html><html lang='en'><head><meta charset='UTF-8'><title>MGR - About</title><link rel='stylesheet' type='text/css' href='/assets/css/style.css?1737135383799963129'><link id='mobile-fonts'><link rel='shortcut icon' href='/assets/images/branding/favicon.svg' type='image/x-icon'><meta http-equiv='cache-control' content='no-cache, no-store, must-revalidate'><meta http-equiv='pragma' content='no-cache'><meta http-equiv='expires' content='0'><meta name='viewport' content='width=device-width'><meta name='google-site-verification' content='Zw2VRjNAyfT-FjfkJqfffHmaHKcHTJ54Dy28j98IYjY'/><link media='(min-width: 900px)' href='https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700' rel='stylesheet'></head><body class='about-page' tabindex='-1'><div class='content'><div class='container'> <header style="padding-left:15px;box-sizing:border-box"><a id='cube-container' href='/#'><div id='cube' aria-label='Yale DHLab logo'><div id='cube-face-a' class='row justify-center align-center'> <img src='/assets/images/branding/IMPERIAL_logo_RGB_Black_2024.svg'/></div><div id='cube-face-b' class='row justify-center align-center'> <img src='/assets/images/branding/mgr_logo.png' style='width:100px;height:auto'/></div></div> </a><div class='nav-links-container' role='navigation' style="margin-left:-10px"><div class='navicon'> <svg class='navicon-icon' viewBox='0 0 1792 1792' xmlns='http://www.w3.org/2000/svg'> <path d='M1664 1344v128q0 26-19 45t-45 19h-1408q-26 0-45-19t-19-45v-128q0-26 19-45t45-19h1408q26 0 45 19t19 45zm0-512v128q0 26-19 45t-45 19h-1408q-26 0-45-19t-19-45v-128q0-26 19-45t45-19h1408q26 0 45 19t19 45zm0-512v128q0 26-19 45t-45 19h-1408q-26 0-45-19t-19-45v-128q0-26 19-45t45-19h1408q26 0 45 19t19 45z'/> </svg></div><div class='nav-links'><div class='nav-links-mobile'> <span class='close-mobile-header'>✕</span></div><div class='link projects'> <a href='/about.html' role='menuitem' aria-label='about'>About</a></div><div class='link projects'> <a href='/team.html' role='menuitem' aria-label='team'>The Team</a></div><div class='link projects'> <a href='/contact.html' role='menuitem' aria-label='contact'>Contact Us</a></div></div></div> </header><div class='hero background-image hero background-image' style='background-image:url("/assets/images/banners/soil_bag.svg")' alt='About Our Mission'><div class='hero-text-container'><div class='card-title-container'><div class='card-title-blur'></div><div class='card-title-text'><div class='small-uppercase'></div><div class='card-title'><b>About Our Mission</b></div></div></div><div class='hero-caption'> Soil collections being prepared for geochemical and metagenomic analysis. Photo by Imperial College London.</div></div></div><div class='center-column'><div class='two-column-container one-third-width top-text'><div class='right-column' role='main'><h1>Our Mission</h1><p>The Microbiome Green Revolution project was developed to tackle a key challenge of the 21st century: how to maintain or increase agricultural productivity while reducing the usage of pesticides and fertilisers, as well as its carbon footprint.</p><p>The solution we have proposed is to harness the trillions of microbes in soil, many of which are beneficial to plants. In particular, the project aims to identify and engineer the soil microbial communities that confer protection against disease. We are focusing on a globally important crop, wheat, and the fungal pathogen that causes Take-All disease (Gaeumannomyces tritici, Gt), which causes significant reductions in yield (typically >10-20 per cent) in continuous wheat.</p><p style="color:#589a90">We sincerely thank the Michael Uren Foundation for their generous support in advancing this pioneering research programme. By utilising an enhanced understanding of microbial ecosystems, we aim to foster sustainable agricultural practices and provide farmers with innovative tools to improve crop yields while preserving biodiversity.</p><h1>Our Research</h1><p>The research has been designed as an interdisciplinary pipeline that combines state-of-the-art modelling and empirical tools. Figure 1 presents the project’s framework illustrating the connections among the tasks along with staff contributions. Specifically: Task 1 will collect agricultural soils from across the UK to create a biobank of soil microbes; Task 2 will perform high-throughput laboratory experiments to identify protective microbial communities and predict their membership; Task 3 will trial these protective communities in the field. These tasks are integrated using bioinformatic tools and theoretical modelling. Cross-cutting tasks involve development of the computational pipelines and theoretical modelling that will be used to interpret the datasets.</p> <figure class="figure-container"> <img src="assets/images/about/tasks.png" alt="Descriptive Alt Text"> <figcaption>Framework of the three main project tasks.</figcaption> </figure><h2 style="color: #be924e;">Task 1: Build a biobank from take-all suppressive soils</h2><p>Our vision is to create a biobank of soils collected from farms across the UK. This biobank would contain the living microbes, kept at cold (4C) or frozen conditions (-80C) to maintain them over the long term. This biobank would be used for the current project and could also be a resource for future projects, for example looking at other plant diseases.</p><p>We have finished our first major field campaign, having collected a total of 696 soil samples from across wheat-growing areas of England (see map Figure 2). We have visited farms directly and sent out soil sampling kits for farmers to return by mail. Wheat crops have been harvested over the past few weeks, which has created a natural break in our sampling. We will extend this campaign in the new year.</p><p>As part of this work, we have also established collaborations with two leading UK research institutions. Rothamsted Research (rothamsted.ac.uk) is a world-renowned agricultural research institution who will aid with the characterisation of Gt. CABI (cabi.org) is an UN-charter research organisation focused on biocontrol of agricultural pathogens in the developing world. They hold the National Fungal Collection, and a cryo-bank of soils and bacteria associated with UK crops and will assist with characterisation and preservation of the soils</p><h2 style="color: #be924e;">Task 2: Artificial selection of disease-suppressive microbiomes</h2><p>We will use the Task 1 biobank to conduct a series of experiments that artificially select for the most protective microbial community. Our goal behind using artificial selection is to accelerate the discovery of optimal protocols for self-assembly of beneficial (protective) microbial communities.</p><p>We are currently conducting the preliminary experiments required to identify appropriate growing conditions and trial measurement techniques, conducted in our Controlled Environment facility (to be completed in 2024). The main experiments will then begin in Winter/Spring 2025.</p> <figure class="figure-container"> <img src="assets/images/about/substrate_experiment.png" alt="Descriptive Alt Text"> <figcaption>Pilot experiment to check which substrates are most appropriate for wheat growth.</figcaption> </figure><h2 style="color: #be924e;">Task 3: Test optimised microbiome assembly protocols in the field</h2><p>The soil microbes and community compositions identified in Tasks 1-2 will be used in a field trial to test their efficacy. This task will be initiated in 2025 following the completion of the first phase of experiments in Task 2.</p><p>We have initiated conversations with farmers and are using information collected in the farmer surveys to identify 0best-practice for continuous wheat cropping with minimal risks of Take-all disease.</p><h2 style="color: #be924e"> Development of computational pipelines (cross cutting task)</h2><p>We are developing robust and reproducible bioinformatic analysis pipelines that will monitor the diverse microbial populations found in the soil samples. Changes in the microbial communities will be compared to wheat yield and resilience to Gt. We are also designing two additional analysis pipelines which will answer:</p><ul><li>Which are the keystone microbes that shape the outcome of the wheat-fungus interaction?</li><li>Which genes help microbes become beneficial symbiotic partners?</li><li>Do viruses play a role in protecting wheat plants from fungal infection?</li></ul><h2 style="color: #be924e"> Future plans development of theory (cross-cutting task)</h2><p>We are developing the mathematical framework for predicting the incidence of Gt in soils once the datasets are complete. The first component involves statistical modelling of the empirical data collected from the field survey (to be completed by Spring 2025). The second component involves building a ‘digital microbiome’ model of the soil microbiomes (to be completed by end of 2024).</p></div></div></div></div></div> <footer role='content-info'><div class='footer-logo'> <a href='http://microbiomegreenrevolution.org' target='_blank'> <img src='/assets/images/branding/mgr_logo_white.png' alt='Digital Humanities Footer Logo'> </a></div><div class='footer-content'><div>Department of Life Sciences, Silwood Park</div><div>Imperial College London</div><div>Buckhurst Rd, Berks SL5 7PY, UK</div><div> <a href='mailto:'></a></div></div> </footer><script src='/assets/bundle.js?1737135383799963129'></script><link media='(max-width: 900px)' href='https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700' rel='stylesheet'></body></html>